lib/MySQL_Variables.cpp

```diff
@@ -0,0 +1,358 @@
+#include "MySQL_Variables.h"
+#include "proxysql.h"
+
+#include "MySQL_Session.h"
+#include "MySQL_Data_Stream.h"
+#include "SpookyV2.h"
+
+#include <sstream>
+
+MySQL_Variables::MySQL_Variables(MySQL_Session* _session) {
+	assert(_session);
+	session = _session;
+	is_connected_to_backend = false;
```

Comment:
Possible usage of member initialization list.

Reply:
Fixed